poppy/fresnel.py

```diff
@@ -129,13 +129,16 @@ def __str__(self):
 
 
 class FresnelWavefront(Wavefront):
+    angular_coordinates = False
+    """Should coordinates be expressed in arcseconds instead of meters at the current plane? """
```

Comment:
Hmm, I generally try to avoid class attributes since they rise confusion and error if you use anything mutable there, and don't realize that it's shared among all class instances, but I suppose this will work properly with a bool.  Still I am not sure it's any better than assigning it in `__init__`...?